.ci.yaml

```diff
@@ -36,7 +35,7 @@ platform_properties:
       os: Windows
 
 targets:
-  - name: Windows win32-platform_tests master
+  - name: Windows win32-platform_tests main
```

Comment:
These name changes should all be reverted; they are referring to Flutter channels.

Reply:
Done, reverted the names back to master.